src/main/java/com/datastax/cdm/properties/KnownProperties.java

```diff
@@ -85,10 +85,10 @@ public enum PropertyType {
         required.add(ORIGIN_KEYSPACE_TABLE);
            types.put(ORIGIN_TTL_NAMES, PropertyType.STRING_LIST);
            types.put(ORIGIN_TTL_AUTO, PropertyType.BOOLEAN);
-        defaults.put(ORIGIN_TTL_AUTO, "true");
+        defaults.put(ORIGIN_TTL_AUTO, "false");
            types.put(ORIGIN_WRITETIME_NAMES, PropertyType.STRING_LIST);
            types.put(ORIGIN_WRITETIME_AUTO, PropertyType.BOOLEAN);
-        defaults.put(ORIGIN_WRITETIME_AUTO, "true");
+        defaults.put(ORIGIN_WRITETIME_AUTO, "false");
```

Comment:
@arvy I believe you may be right here. For example, if we have a counter table, then we receive the following when it is turned on,
```
2023-12-15 15:45:31 ERROR [] WritetimeTTL:117 - Counter table cannot specify TTL or WriteTimestamp columns as they cannot set on write
2023-12-15 15:45:31 ERROR [] CopyJobSession:80 - Feature com.datastax.cdm.feature.WritetimeTTL is not valid.  Please check the configuration.
```